front/tests/008-train-schedule.spec.ts

```diff
@@ -0,0 +1,109 @@
+import { test } from '@playwright/test';
+import { v4 as uuidv4 } from 'uuid';
+
+import type { Infra, Project, Scenario, Study, TimetableResult } from 'common/api/osrdEditoastApi';
+
+import scenarioData from './assets/operationStudies/scenario.json';
+import HomePage from './pages/home-page-model';
+import OperationalStudiesTimetablePage from './pages/op-timetable-page-model';
+import ScenarioPage from './pages/scenario-page-model';
+import { readJsonFile } from './utils';
+import { getInfra, getProject, getStudy, postApiRequest } from './utils/api-setup';
+import { postSimulation, sendTrainSchedule } from './utils/trainSchedule';
+
+let selectedLanguage: string;
+let smallInfra: Infra;
+let project: Project;
+let study: Study;
+let scenario: Scenario;
+let timetableResult: TimetableResult;
+
+const trainSchedulesJson = readJsonFile('./tests/assets/trainSchedule/train_schedules.json');
+test.beforeAll(async () => {
+  // Fetch infrastructure, project, study, and timetable result
+  smallInfra = (await getInfra()) as Infra;
+  project = await getProject();
+  study = await getStudy(project.id);
+  timetableResult = await postApiRequest(`/api/v2/timetable/`);
+
+  // Create a new scenario with a unique name
+  scenario = await postApiRequest(`/api/v2/projects/${project.id}/studies/${study.id}/scenarios/`, {
+    ...scenarioData,
+    name: `${scenarioData.name} ${uuidv4()}`,
+    study_id: study.id,
+    infra_id: smallInfra.id,
+    timetable_id: timetableResult.timetable_id,
+  });
+
+  // Post train schedule import and simulation
+  const response = await sendTrainSchedule(scenario.timetable_id, trainSchedulesJson);
+  await postSimulation(response, smallInfra.id);
```

Comment:
The postSimulation is only used in these tests and needs to be run just once in `beforeAll`
The infra is loaded in `global-setup.ts`, in this test, we only use the ID of the loaded infra

Reply:
Why do you need to run the postSimulation in the beforeAll ? Is it to cache the result in the backend ? If that's the case, a comment to explain it would be nice :)